src/main/java/de/refactoringbot/services/main/RefactoringService.java

```diff
@@ -164,49 +167,55 @@
 		List<RefactoredIssue> allRefactoredIssues = new ArrayList<>();
```

Comment:
The method parameter `amountBotRequests` can now be removed from the signature.

Reply:
Done.